.github/workflows/release.yml

```diff
@@ -0,0 +1,9 @@
+on: create
+
+name: Release
+jobs:
+  release:
+    name: Release GitHub Actions
+    runs-on: ubuntu-latest
+    steps:
+      - uses: technote-space/release-github-actions@v6
```

Comment:
This doesn't take a tag or version number. How does it know what to release?

Reply:
I updated README, this will run when you create and push a git tag manually. It will keep major and minor tags in sync. Happy to change how this works, but I've noticed a convention for GHA that `v1` means latest `v1.*.*` release.

ex: `uses: dbt-labs/actions/parse-semver@v1`

Reply:
Gotcha! No that should be perfect. Thanks Kyle